t/features/step_definitions/analizo_steps.pl

```diff
@@ -153,6 +153,22 @@
   }
 };
 
+Then qr/^analizo must report that module (.+) has not (.+) = (.+)$/, sub {
+  my ($c) = @_;
+  my ($module, $metric, $value) = ($1, $2, $3);
+  my @stream = Load($stdout);
+  my ($module_metrics) = grep { $_->{_module} && $_->{_module} eq $module } @stream;
+  if ($module_metrics->{$metric}) {
+    if ($value =~ /^\d+|\d+\.\d+$/) {
+      cmp_ok($module_metrics->{$metric}, '!=', $value);
+    }
+    elsif ($value =~ /^\[(.*)\]$/) {
+      my @values = split(/\s*,\s*/, $1);
+      is_deeply($module_metrics->{$metric}, \@values);
```

Comment:
I suppose that this comparison don't do the right thing. See `perldoc Test::More` for details about the `is_deeply()` function.

Reply:
Thanks, this is something to improve, i let this comparison pass by. I only copied the rule above and changed the '==' to '!='. Our goal was to use this rule to check if some metrics that should have some value, were not empty.